specification/eventgrid/data-plane/Microsoft.DataBox/stable/2018-01-01/DataBox.json

```diff
@@ -0,0 +1,74 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Data Box events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure Data Box events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "DataBoxCopyStartedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.DataBox.CopyStarted event.",
+      "type": "object",
+      "properties": {
+        "serialNumber": {
+          "description": "Serial Number of the device associated with the event, separated by comma delimiter in case of multiple",
+          "type": "string"
+        },
+        "stageDetails": {
+          "$ref": "#/definitions/StageDetail"
+        }
+      }
+    },
+    "DataBoxCopyCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.DataBox.CopyCompleted event.",
+      "type": "object",
+      "properties": {
+        "serialNumber": {
+          "description": "Serial Number of the device associated with the event, separated by comma delimiter in case of multiple",
+          "type": "string"
+        },
+        "stageDetails": {
+          "$ref": "#/definitions/StageDetail"
+        }
+      }
+    },
+    "DataBoxOrderCompletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.DataBox.OrderCompleted event.",
+      "type": "object",
+      "properties": {
+        "serialNumber": {
+          "description": "Serial Number of the device associated with the event, separated by comma delimiter in case of multiple",
+          "type": "string"
+        },
+        "stageDetails": {
+          "$ref": "#/definitions/StageDetail"
+        }
+      }
+    },
+    "StageDetail": {
```

Comment:
Copying this comment from the initial PR https://github.com/Azure/azure-rest-api-specs/pull/21157/

Are there a set of well known stages? Or is this totally free form?

If there are a set of stages that may expand in the future, we should do something like this:
https://github.com/Azure/azure-rest-api-specs/blob/a21a67abf47640c632ed3440cd76ee750806398b/specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json#L477-L483